### ISSUE TITLE
Subsequently defined placeholders clears previously defined placeholders?

### ISSUE THREAD

**Issue by jenshult** [jamesramsay/hercule#96]
File *test.md* sets a string placeholder to "venues":
```
:[](blueprints/venues.md resource:"venues")
```
File *blueprints/venues.md*
```
# Group Venues

## Add venues [/venues]

:[](post.md attributes:"Venue" description:"This and that")
```
File *post.md* is expected to just print the *resource* placeholder
```
:[](resource)
```
Now, transcluding *test.md* gives me the following error:
```
$ hercule --verbose doc/test.md -o doc/out/APIv2local.apib
Transcluding file... doc/test.md
Links found: 1
Transcluding: doc/blueprints/venues.md (file) into doc/blueprints/venues.md
Links found: 1
Transcluding: doc/blueprints/post.md (file) into doc/blueprints/post.md
Links found: 1
Error: File (doc/blueprints/resource) not found.
Transcluding: doc/blueprints/resource (file) into doc/blueprints/resource
Links found: 0
```
If I remove the placeholders in the second file, then all is well. 
It seems to me, that subsequently defined placeholders will clear all previously defined placeholder during the transclusion, or am I just doing it wrong? 

If this is a chosen or known limitation, then the possibility to merge placeholders would be a really nice feature.

**Comment by jamesramsay** [comment]
@jenshult it's meant to be merging. Thanks for the really clear steps to reproduce. :+1:

I'll try to have this fixed soon!

**Comment by jamesramsay** [comment]
Fixed in v1.2.7.

**Comment by jenshult** [comment]
@jamesramsay Work like a charm on my end! Thanks for the super speedy fix and release!
